warehouse/templates/manage/token.html

```diff
@@ -35,7 +35,9 @@ <h2>Token for "{{ macaroon.description }}"</h2>
       {% if macaroon.caveats.permissions == "user" %}
       <strong>Scope:</strong> Entire account (all projects)
       {% else %}
-      <strong>Scope:</strong> Project "{{ macaroon.caveats.permissions.projects[0] }}"
+      {% for project in macaroon.caveats.permissions.projects %}
+        <strong>Scope:</strong> Project "{{ project }}"<br>
```

Comment:
nit-pick: We probably shouldn't add a "Scope:" for every project and instead have it just once, with the project names in a comma-separated-with-and-for-final-join manner.